static/scss/review.scss

```diff
@@ -0,0 +1,62 @@
+.review-card {
+
+  &.review-panel-right {
+    border-left: 1px solid black;
+  }
+
+  .status {
+    width: 75%;
+    margin: auto;
```

Comment:
I'm not sure we need these styles. If anything you can set a left padding with a percentage if we really need to keep this form padded from the divider